runtime/libpgmath/lib/common/amod.c

```diff
@@ -7,11 +7,11 @@
 
 #include "mthdecls.h"
 
-#ifndef WIN64
+#ifndef TARGET_WIN_X8664
 float
 __mth_i_amod(float f, float g)
 {
-#if defined(WIN64)
+#if defined(TARGET_WIN_X8664)
```

Comment:
This block is never compiled, due to the outer `#ifndef TARGET_WIN_X8664`. Could you just delete it?

Reply:
looks like it was disabled in 6bc8b5aa55b16afdf0d53fcca024b8aa3bfb4814. Interesting.